packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3509,30 +3509,31 @@ export default class Meeting extends StatelessWebexPlugin {
   /**
    * Manages be right back status updates for the current participant.
    *
-   * @param {boolean} enabled - Indicates whether the use is brb or not.
-   * @returns {Promise<void>} - A promise that resolves when the request is complete.
+   * @param {boolean} enabled - Indicates whether the user enabled brb or not.
+   * @returns {Promise<void>} resolves when the brb status is updated or does nothing if not in a multistream meeting.
    * @throws {Error} - Throws an error if the request fails.
-   *
-   * @todo non-multistream support https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-578667
    */
-  public beRightBack(enabled: boolean) {
-    return this.meetingRequest
-      .sendBrb({
-        enabled,
-        locusUrl: this.locusUrl,
-        deviceUrl: this.deviceUrl,
-        selfId: this.selfId,
-      })
-      .then(() => {
-        if (this.isMultistream && this.mediaProperties.webrtcMediaConnection) {
+  public beRightBack(enabled: boolean): Promise<void> {
+    // this logic should be applied only to multistream meetings
+    if (this.isMultistream && this.mediaProperties.webrtcMediaConnection) {
+      return this.meetingRequest
+        .sendBrb({
+          enabled,
+          locusUrl: this.locusUrl,
+          deviceUrl: this.deviceUrl,
+          selfId: this.selfId,
+        })
+        .then(() => {
           this.sendSlotManager.setSourceStateOverride(MediaType.VideoMain, enabled ? 'away' : null);
-        }
-      })
-      .catch((error) => {
-        LoggerProxy.logger.error('Meeting:index#beRightBack --> Error ', error);
+        })
+        .catch((error) => {
+          LoggerProxy.logger.error('Meeting:index#beRightBack --> Error ', error);
 
-        return Promise.reject(error);
-      });
+          return Promise.reject(error);
+        });
+    }
+
+    return undefined;
```

Comment:
Do we want to return undefined here or do we want to return an empty promise? Maybe we can omit the return statement entirely? I'm a bit surprised we're allowed to return undefined here without TypeScript complaining, since we don't specify undefined can be returned in the function signature.

Reply:
I am more inclined to have an empty promise here, though @antsukanova has a point to have it as undefined as other functions have similar return statements.

Reply:
If we made this method `async`, no return statement will be required

Reply:
@brycetham, as discussed, let's then do throwing error way to handle transcoded flow. @edvujic, could you please support and change this part alongside with test cases pls? Appreciate that